src/hexo.ts

```diff
@@ -0,0 +1,310 @@
+const draftGenerator: Fig.Generator = {
+  script: "hexo list post | grep -E ^Draft",
+  postProcess: (out) => {
+    return out.split("\n").map(function (file) {
+      const title = file
+        .replace(/^.*   *_drafts\//g, "")
+        .replace(/  *(.*)$/g, "")
+        .replace(/.md$/g, "");
+      return {
+        name: title,
+        description: "draft for " + title,
```

Comment:
Just a quick case change 🙂 (we try to keep to "conventional descriptions", eg. upper case first letters, no periods at the end)
```suggestion
        description: "Draft for " + title,
```

Reply:
Gotcha, and updated the description